src/Microsoft.Health.Fhir.Store.Utils/BatchExtensions.cs

```diff
@@ -0,0 +1,138 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Store.Utils
+{
+    public static class BatchExtensions
```

Comment:
Any reason this class file and the CancelRequest.cs file are placed in a different folder? Unless the plan is to reuse them, including them in the import project folder would make it easy to download and test the import tool.

Reply:
Yes, I plan to use BatchExtentions on other project too.